tests/integration/security/authz_test.go

```diff
@@ -110,6 +110,76 @@ func TestAuthz_Principal(t *testing.T) {
 		})
 }
 
+func TestAuthz_ServiceAccount(t *testing.T) {
+	framework.NewTest(t).
+		Run(func(t framework.TestContext) {
+			allowed := apps.Ns1.A
+			denied := apps.Ns2.A
+
+			from := allowed.Append(denied)
```

Comment:
Could you explain why we're appending the allowed and denied namespaces here?

Reply:
FWIW this is copied from the test above. But I think this is basically saying "We want to send requests from all relevant workloads". Relevant workloads are those that have policies about them

Reply:
Ah `from` refers to the echo config and not the AuthzPolicy

Reply:
Ahh gotcha, I was thinking it for the AuthzPolicy. Makes sense now. Thanks!